environment.yml

```diff
@@ -10,6 +9,7 @@ dependencies:
   - opensimplex == 0.4.5
   - numpy >=1, < 2
   - pydantic >=2, <3
+  - pip
```

Comment:
Readthedocs was complaining about not having an explicit pip dependency in its logs.